framework/db/src/com/cloud/utils/db/TransactionLegacy.java

```diff
@@ -1014,6 +1014,13 @@ public static void initDataSource(Properties dbProps) {
             if (dbProps.size() == 0)
                 return;
 
+            try {
+                // Explicitely load JDBC driver because it has been removed from the classpath as part of commit c22659d76d73f00f41c13776c490e17a50aacd20
+                Class.forName("com.mysql.jdbc.Driver");
```

Comment:
two questions:
hardcoded driver name?; now that we can shouldn't we prepare for a MariaDB driver to be loaded instead?
if it is not on the classpath how is this going to help, then? Class.forName(..) will search the classpath